Cargo.toml

```diff
@@ -67,8 +67,8 @@ members = [
 
 [workspace.lints.clippy]
 cast_lossless = "deny"
-complexity = "deny"
-correctness = "deny"
+complexity = { level = "deny", priority = -1 }
```

Comment:
New clippy warnings for these on nightly.

Reply:
What does the priority  = -1 do?

Reply:
It overrides the group / collection lints. https://rust-lang.github.io/rust-clippy/master/index.html#/lint_groups_priority